indico/web/client/styles/partials/_inputs.scss

```diff
@@ -149,10 +149,6 @@ ul {
 
 div[contenteditable],
 span[contenteditable] {
```

Comment:
I wonder if we need *any* of that contenteditable styling oonsidering that we no longer use it ourselves anywhere in the codebase and ckeditor should have its own internal styling...

Reply:
Yeah, those were mostly unused. There was a dashed border on hover, but I don't think this is necessary.

Reply:
in fact the dashed border on hover looked weirdt here but was so subtle that probably nobody noticed it. this whole style was meant for another place where we now actually have a simple textarea for plaintext+markdown (in the past we used contenteditable for registration form sections, probably to allow people to paste formatted content from elsewhere - with all the disadvantages that come with that)